### PR TITLE
feat: add `/vm.m_run` message handler

### PR DESCRIPTION
**Pull request by luciorubeens**
Add handlers for the new message type: https://github.com/gnolang/gno/pull/1001

### CODE REVIEW

File: packages/adena-module/src/transaction/message-info/message-info-vm.ts [hunk 1]
```diff
@@ -80,3 +86,16 @@ export const encodeVmAddPackage = (writer: protobuf.Writer, messageInfo: VmAddPa
   }
   return writer;
 };
+
+export const encodeVmRun = (writer: protobuf.Writer, messageInfo: VmAddPackage) => {
```
Comment (r3v4s) [inline packages/adena-module/src/transaction/message-info/message-info-vm.ts:90]: shouldn't it be `messageInfo: VmRun` ??

**Comment by luciorubeens** [comment]
I tried to broadcast a transaction, but I got this error, can you help to investigate?

CleanShot 2023-10-19 at 17 12 51@2x

File: packages/adena-module/src/transaction/message-info/message-info-vm.ts [hunk 2]
```diff
@@ -80,3 +86,16 @@ export const encodeVmAddPackage = (writer: protobuf.Writer, messageInfo: VmAddPa
...
+
+export const encodeVmRun = (writer: protobuf.Writer, messageInfo: VmRun) => {
+  if (messageInfo.caller !== '') {
+    writer.uint32(10).string(messageInfo.caller);
+  }
+  if (messageInfo.send !== '') {
+    writer.uint32(18).string(messageInfo.send);
+  }
+  if (messageInfo.package !== undefined) {
+    encodePackage(writer.uint32(18).fork(), messageInfo.package).ldelim();
```
Comment (jinoosss) [inline packages/adena-module/src/transaction/message-info/message-info-vm.ts:98]: This simply seems to be the problem.
Change the size of the buffer in the 3rd argument.
`18` -> `26`

Comment (r3v4s) [reply packages/adena-module/src/transaction/message-info/message-info-vm.ts:98]: @luciorubeens seems like protobuf has certain buf size value for nth args.
https://www.tabnine.com/code/javascript/functions/protobufjs/Writer/string

Comment (luciorubeens) [reply packages/adena-module/src/transaction/message-info/message-info-vm.ts]: Perfect, thank you guys, now it works

CleanShot 2023-10-20 at 10 00 54@2x

**Review by jinoosss — APPROVED** [review]
@luciorubeens 
Thank you for your work.

**Review by r3v4s — APPROVED** [review]
looking good

**Pull request merged**